docs/Manuals/Program/freeciv21-modpack-qt.rst

```diff
@@ -0,0 +1,56 @@
+..  SPDX-License-Identifier: GPL-3.0-or-later
+..  SPDX-FileCopyrightText: James Robertson <jwrober@gmail.com>
+
+freeciv21-modpack-qt
+********************
+
+SYNOPSIS
+========
+
+``freeciv21-modpack-qt`` [ -i|--install `<URL>` ] [ -L|--List `<URL>`] [ -h|--help ] [ -v|--version ]
+
+DESCRIPTION
+===========
+
+.. include:: freeciv21-server.rst
+  :start-line: 17
+  :end-line: 29
+
+This is a program to read and install a modpack URL from the command line. When given no command line
+parameters the Qt base GUI will load and allow for modpack installation. For a pure command line modpack
+installer, see freeciv21-modpack(6).
+
+OPTIONS
+=======
+
+The following options are accepted on the command line of the manual generator. They may not be combined as
```

Comment:
I'm guessing "manual generator" in this sentence should be "modpack installer"?

Reply:
Oh good catch. I copied the -manual.rst page.